README.md

```diff
@@ -35,6 +33,11 @@ You can find an example of using this integration on [our website](https://dev.f
 
 You can find more info on the FingerprintJS Pro agent documentation page [https://dev.fingerprint.com/docs/js-agent](https://dev.fingerprint.com/docs/js-agent)
 
+`Tag type` - The way you want to use the tag. It is 3 options:
+  - `Load and identify` – default behavior. Load JS Agent and instantly identify browser.
+  - `Load` – just load JS Agent.
+  - `Identify` - identify browser. JS Agent needs to be loaded before. You can collect necessary data, then trigger this tag and provide `linkedId` or `tag`.
```

Comment:
Yes `Identify` tag requires a `Load` tag. Load JS agent programmatically will not work because GTM tag requires special wrapper.

But if the customer loaded Agent using `Load` or `Load and identify` tag, it is possible to use the agent programmatically without adding a script tag.

Just as in documentation snippet:

```js
var fpPromise = FingerprintJS.load()

  // Analyze the visitor when necessary.
  fpPromise
    .then(function (fp) { return fp.get() })
    .then(result => console.log(result.requestId, result.visitorId))
```